pallets/phala-world/src/incubation.rs

```diff
@@ -488,6 +488,10 @@ pub mod pallet {
 					("name", property_value(&part_info.name)),
 					("slot_name", slot_name_value.clone()),
 					("rarity", property_value(&part_info.rarity)),
+					("race", property_value(&part_info.rarity)),
```

Comment:
```suggestion
					("race", property_value(&part_info.race)),
```
Also, check comment below since the `race` and `career` are already added in line 429-430